### DISCUSSION TITLE
Lazer immediately crashes on iOS at startup

### DISCUSSION THREAD

**Discussion by 59xa** [ppy/osu#15996] [Q&A] [answered]
It is good to see the app back on TestFlight for testing, however, it is not good to see the startup crashing a second later after opening the client. Upon startup the osu! logo does not appear, nor the intro animation. I’ve provided logs, but I’m not sure if these will help.

**Platform:** iOS 15.1 on an iPhone 11
**How to reproduce:** Just start the app.
**Version:** 2021.1207.0 

[Crash Logs for 2021.1207.0 iOS Testflight](https://mega.nz/folder/pXIzQSQC#fcILLZpz7pWJJX4RB4ZELA)

**Comment by frenzibyte** [comment]
Couldn't find anything interesting in the crash logs you attached. Did osu! generate any logs in the application folder? (see https://github.com/ppy/osu/discussions/15188 for their location on iOS).

**Reply by 59xa** [reply]
I would pull the logs for you from that directory right now, but I have no access to any computers of sort. That's all I can provide with this issue, sorry. Someone with a computer with iTunes installed would have to do it for me as I can not provide these crashing issues.

**Reply by jechosu** [reply]
I have successfully pulled the logs, but this is on an iPhone 8 Plus. Hope this is helpful.
[runtime.log](https://github.com/ppy/osu/files/7675797/runtime.log)

**Reply by 59xa** [reply]
This is the correct crash log we are supposed to look for, thanks a bunch.

**Reply by frenzibyte** [reply]
Yep thanks, while it's not quite clear what the offending code line is, at least I have a stack trace that I could start investigating at, for the time being.

**Reply by jechosu** [reply]
Your welcome!

**Comment by frenzibyte** [comment, marked as answer]
The issue has now been identified and fixed, and osu! shall see an iOS build next release.

**Reply by 59xa** [reply]
Looking forward to the return of osu! on TestFlight, cheers!

**Reply by jechosu** [reply]
Can't wait till it comes out!
